pdfminer/layout.py

```diff
@@ -522,13 +563,97 @@ def __init__(self, bbox):
         self.groups = None
         return
 
-    # group_objects: group text object to textlines.
-    def group_objects(self, laparams, objs):
+    def hsplitted(self, obj0, obj1, splitobjs, cell_margin):
```

Comment:
Improve method signature. E.g. `is_separated_by_any(obj0, obj1, other_objects, margin)` or something better.

Reply:
I have gone ahead and changed the naming per your request.  What are your thoughts?

Reply:
I think changing it to `is_splitted_horizontally()` most clearest

Reply:
Excellent!  Change has been incorporated!